### PR TITLE
Update TXT record for huygd.is-a.dev

### PR DESCRIPTION
**Pull request by huygmdvn**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] The CNAME record doesn't contain `https://` or `/`.  
- [x] There is sufficient information at the `owner` field.

### CODE REVIEW

**Comment by huygmdvn** [comment]
Register a new domain and Up a domain

**Comment by Satindar31** [comment]
Merge conflicts have to be fixed

File: domains/huygd.json [hunk 1]
```diff
@@ -7,7 +7,10 @@
   },
   "record": {
     "A": [
-      "103.77.240.59"
+      "103.77.240.59",
```
Comment (Stef-00012) [inline domains/huygd.json:10]: remove this comma
```suggestion
      "103.77.240.59"
```

File: domains/huygmd.json [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{
+  "description": "The domain I will use to test a PHP code",
+  "repo": "https://github.com/huygmdvn/register",
+  "owner": {
+    "username": "huygmdvn",
+    "email": "huynhquochuytkna@gmail.com"
+  },
+  "record": {
+    "A": [
+      "103.77.240.59",
```
Comment (Stef-00012) [inline domains/huygmd.json:10]: remove this comma
```suggestion
      "103.77.240.59"
```

**Review by Stef-DP — CHANGES_REQUESTED** [review]
please resolve the merge conflict

File: domains/huygd.json [hunk 3]
```diff
@@ -7,7 +7,10 @@
...
+    ]
```
Comment (Stef-00012) [inline domains/huygd.json:11]: add a comma here
```suggestion
    ],
```

File: domains/huygmd.json [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    ]
```
Comment (Stef-00012) [inline domains/huygmd.json:11]: add a comma here
```suggestion
    ],
```

**Review by Stef-DP — CHANGES_REQUESTED** [review]

**Comment by huygmdvn** [comment]
> please resolve the merge conflict

Have resolve,can you check it again?

**Comment by Satindar31** [comment]
> > please resolve the merge conflict
> 
> Have resolve,can you check it again?

The merge conflicts are still there. If you don't know how to resolve them, the easiest way is to probably just delete your fork, refork it, make the changes and then make a new pr

**Pull request closed without merging**